commit d53b9cc391c72a1011ea8fe7a9f70dc5060a0db2
Author: John David Anglin <danglin@gcc.gnu.org>
Date:   Tue Mar 15 23:04:39 2022 +0000

    hppa: Use END instead of PSEUDO_END in swapcontext.S
    
    (cherry picked from commit 7a5c440102d4ec7fafd9bbd98eca9bd90ecaaafd)

diff --git a/sysdeps/unix/sysv/linux/hppa/swapcontext.S b/sysdeps/unix/sysv/linux/hppa/swapcontext.S
index 94b164dc6375563e..fbc22586d1195a0d 100644
--- a/sysdeps/unix/sysv/linux/hppa/swapcontext.S
+++ b/sysdeps/unix/sysv/linux/hppa/swapcontext.S
@@ -67,6 +67,6 @@ ENTRY(__swapcontext)
 	ldw oR28(%ret1),%rp
 	bv,n %r0(%rp)
 
-PSEUDO_END(__swapcontext)
+END(__swapcontext)
 
 weak_alias (__swapcontext, swapcontext)
